src/test/scala/mesosphere/marathon/integration/setup/MarathonCallbackTestSupport.scala

```diff
@@ -81,4 +85,15 @@ trait MarathonCallbackTestSupport extends ExternalMarathonIntegrationTest {
 
     receivedEventsForKinds.groupBy(_.eventType)
   }
+
+  def extractDeploymentIds(app: RestResult[AppDefinition]): Seq[String] = {
```

Comment:
This call is related to the MarathonAPI. So you could for example put it into its own class in `mesosphere.marathon.integration.setup` or maybe as part of the MarathonFacade?